protocol/README.md

```diff
@@ -45,6 +45,7 @@ In other words, these higher ranges are reserved for use by specific Ably projec
 | From | To | Project / Ecosystem | Location of Definitions |
 | ---- | -- | ------------------- | ----------------------- |
 | 100000 | 100999 | Asset Tracking | [Specification: Error codes](https://github.com/ably/ably-asset-tracking-common/tree/main/specification#error-codes) |
+| 101000 | 101999 | Spaces | [Specification: Error codes](https://github.com/ably-labs/spaces/tree/main/specification#error-codes) |
```

Comment:
Thinking about it, I think we should just put the error codes in this repo, maybe just in `errors.json` with the others? I don't see a reason to maintain them separately tbh.

Reply:
We can do this, but the docs above this table no longer makes sense when it states that:

> Codes from `100000` are defined outside of scope of this repository.
> In other words, these higher ranges are reserved for use by specific Ably projects and ecosystems, where these codes are unknown to the core Ably service

I can change the wording slightly so other SDKs can re-use this repository.

Reply:
I've updated in 5fb8a0434a4a010e599273863a3ae4c102868f68.

I've also included a link to FAQs in `errorsHelp.json`... but I'm guessing this doesn't exist yet. Is this something we need to add?

Reply:
> I'm guessing this doesn't exist yet. Is this something we need to add?

They need to be added but I'm not sure by who, see [this Slack discussion](https://ably-real-time.slack.com/archives/C04B5EKDUKC/p1692878438128949).